packages/core/test-bundles/bundlesize.json

```diff
@@ -2,7 +2,7 @@
   "files": [
     {
       "path": "./dist/core/global.min.css",
-      "maxSize": "7 kB",
+      "maxSize": "8.2 kB",
```

Comment:
The new color palette bumped this up about a kb. Hoping to look into a way to push that back down in a follow up PR.